src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/RedirectController.cs

```diff
@@ -41,7 +44,7 @@ public ActionResult ValidateUrl([BindRequired, FromQuery, Url] string url)
         {
             Uri uri = new Uri(url);
 
-            if (_appSettings.Hostname != uri.Host)
+            if (!IsValidRedirectUri(uri.Host))
```

Comment:
Should we also update the action to define return type?
`ActionResult<string>`
And be specific about return type in the case of a BadRequest?
Not sure how smart SwashBuckle is when generating OpenAPI specification.